mne_bids/path.py

```diff
@@ -540,6 +548,27 @@ def update(self, *, check=None, **kwargs):
                 else:
                     continue
 
+            if key == 'space':
```

Comment:
you mean, such that this whole new "check" is only triggered when the BIDSPath "check" param is True?

That'd be fine by me - please push to this PR! :)

Reply:
I think one more todo would also be to raise a warning if a deprecated coordsystem is being passed. --- it's allowed/accepted by BIDS ... but *shouldn't* be used.

Reply:
ok, nevermind the "push to this PR" --> I did your requested changes.

Also I did not add warnings about deprecated coordsystems ... the bids-validator may be a better place for this: One place to issue such warnings instead of distributing them across each bids tool.